src/test/java/org/codewars/utils/ReaderTest.java

```diff
@@ -30,9 +37,25 @@ public void testReadLong() {
     public void testReadString() {
     }
 
-    @Test
-    public void testReadArrInt() {
+    @DataProvider(name = "testReadArrIntData" )
+    private Object[][] testReadArrIntData(){
+        Object[][] testData = new Object[][]{
+                {"1 3 5 7 9", new int[]{1, 3, 5, 7, 9}},
+                {"12 13 14 22 23 24", new int[]{12, 13, 14, 22, 23, 24}},
+                {"123 234 345", new int[] {123, 234, 345}}
+        };
+        return testData;
     }
+    @Test(dataProvider = "testReadArrIntData")
+    public void testReadArrInt(String input, int[] expected) {
+        InputStream i = new ByteArrayInputStream(input.getBytes());
+        System.setIn(i);
+        Scanner scanner = new Scanner(System.in);
+        Reader reader = new Reader(scanner);
```

Comment:
Reader reader = new Reader();

Reply:
This is the old version, I haven't changed it yet